src/main/java/com/routeme/predictionio/PredictionIOClient.java

```diff
@@ -0,0 +1,114 @@
+package com.routeme.predictionio;
+
+import io.prediction.EventClient;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.concurrent.ExecutionException;
+
+import com.google.common.collect.ImmutableMap;
+import com.routeme.dto.RouteDTO;
+import com.routeme.model.DummyRoute;
+
+public class PredictionIOClient {
+    private final String accessKey = "eJErhxNseKZCZ8ijxy0fLjtj8jeHvb2ngMLoRIpwPGr5inugcbGHGSAgvJM1ZqLs";
+    public static final String PREFERENCE_ROUTE_TYPE_LEASTTIME = "leastTime";
+    public static final String PREFERENCE_ROUTE_TYPE_LEASTCHANGES = "leastChanges";
+    public static final String PREFERENCE_ROUTE_MODE_BUS = "bus";
+    public static final String PREFERENCE_ROUTE_MODE_UBAHN = "ubahn";
+    private EventClient eventClient;
+    Random rand = new Random();
+    Map<String, Object> emptyProperty = ImmutableMap.of();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/heshamhamdymassoud/RouteMe-API/file/2580673877/issues/source?bid=3287637&fileBranchId=3302754#l23)